app/assess/routes.py

```diff
@@ -144,6 +144,32 @@ def display_sub_criteria(
             (1, "Poor"),
         ]
 
+        if request.args.get("add-comment") == "1":
+            display_comment_box = True
+
+        comment_form = CommentsForm()
+
+        if comment_form.validate_on_submit():
+            comment = comment_form.comment.data
+            display_comment_box = False
+
+            submit_comment(
+                comment=comment,
+                application_id=application_id,
+                sub_criteria_id=sub_criteria_id,
+                user_id=g.account_id,
+                theme_id=theme_id,
+            )
+
+            return redirect(
+                url_for(
+                    "assess_bp.display_sub_criteria",
+                    application_id=application_id,
+                    sub_criteria_id=sub_criteria_id,
+                    theme_id=theme_id,
+                )
+            )
+
```

Comment:
I think this is already handled now on line 45-65, No?

Reply:
you're right! have sorted this now 👍🏾